varsfiles/staging.yml

```diff
@@ -0,0 +1,45 @@
+zone-transfer-ips: (( concat terraform_outputs.public_subnet_az1_cidr ", " terraform_outputs.public_subnet_az2_cidr ))
+master-ips: (( join "; " terraform_outputs.ENVIRONMENT_dns_private_ips ))
```

Comment:
If you are going through and doing separate varsfiles like this, no need for the ENVIRONMENT bit, as `sed` should be yanked from processing script as mentioned earlier.